chapters/robots.adoc

```diff
@@ -18,6 +18,22 @@ The <<Referee, referee>> has to force a team to remove a robot from the field if
 ==== Shape
 A robot must fit inside a 0.18 meters wide and 0.15 meters high cylinder at any point in time. Additionally, the top of the robot must adhere to the standard pattern size and surface constraints.
 
+==== Robot's Hull Color
+To ensure clarity and effective identification during gameplay, the following rules apply to the coloring of robot hulls:
+
+Bright and Dark Color Scheme:
+
+* All robots from a team must include two interchangeable colors: one bright and one dark.
+* These two colors must be the same for all robots in a team.
+* The color must cover at least 40% of the robot's maximum height (15 cm), equivalent to a minimum of 6 cm of vertical coverage.
```

Comment:
What, if the robot is smaller? Is the actual robot height, or the allowed robot height considered?

Reply:
I was debating with myself on this. It could be 40% of the robot's actual height. I'm afraid someone comes with a small robot, so the 40% will not be visible. But I don't think we will have robots lower than 10 cm. 😅

Reply:
I'd make it fixed length for simplicity. That's also easier to check ;)

Reply:
Sure! I will fix in 6cm, I think it will be enough to view the robot's difference.